### PR TITLE
- docs for tools.msvs_toolset

### PR DESCRIPTION
**Pull request by SSE4**
docs for https://github.com/conan-io/conan/issues/7061

### CODE REVIEW

File: reference/tools.rst [hunk 1]
```diff
@@ -1747,3 +1747,20 @@ for ``compiler.cppstd=17``, and so on.
 
 Parameters:
     - **settings** (Required): Conanfile settings. Use ``self.settings``.
+
+
+.. _tools.msvs_toolset:
+
+tools.msvs_toolset():
+---------------------
```
Comment (danimtb) [inline reference/tools.rst:1755]:
```suggestion
tools.msvs_toolset()
--------------------
```

File: reference/tools.rst [hunk 2]
```diff
@@ -1747,3 +1747,20 @@ for ``compiler.cppstd=17``, and so on.
...
+
+.. code-block:: python
+
+    def msvs_toolset(settings)
+
+Returns the corresponding Visual Studio platform toolset based on the settings. For instance, it may return ``v142``
+for ``compiler=Visual Studio`` with ``compiler.version=16``. If ``compiler.toolset`` was set in settings, it has a
+priority and always returned.
```
Comment (danimtb) [inline reference/tools.rst:1763]: this means that the tool has no effect and it just returns the ``compiler.toolset`` value, right?

Comment (SSE4) [reply reference/tools.rst:1763]: correct, the only effect is returning `compiler.toolset` value in this case

**Comment by jgsogo** [comment]
Oh! There are conflicting files

**Comment by SSE4** [comment]
how come? I already solved conflicts yesterday. I've checked about 1 hour ago, there were no conflicts!

**Pull request merged**